pkg/webhook/cluster/mutation/mutation.go

```diff
@@ -96,6 +96,15 @@ func (h *AdmissionHandler) Handle(ctx context.Context, req webhook.AdmissionRequ
 }
 
 func (h *AdmissionHandler) applyDefaults(c *kubermaticv1.Cluster) {
+	// Add cni-plugin annotation if not present pointing to latest canal
```

Comment:
actually this does not seem to be adding annotation as the comment suggests, but modifying cluster the spec